TicketsDemo.Data/Entities/Decorators/Price/PriceDecorator.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TicketsDemo.Data.Entities.Decorators
+{
+    public abstract class PriceDecorator : PriceComponent
+    {
+        protected PriceComponent component;
+        public PriceDecorator(PriceComponent component):base()
```

Comment:
please delete it (what's the point of having abstract class inheriting another class?)